services/divshot/README.md

```diff
@@ -0,0 +1,112 @@
+Divshot
+========
+
+Our static site service of choice
+
+How to set up Divshot and CircleCI for a new project
+----------------------------------------------------
+
+### Divshot.io Actions
+
+First thing to do is set up a new Divshot app via the [divshot.io](http://divshot.io) dashboard.
+
+![](http://files-misc.s3.amazonaws.com/create-app.png)
+
+Once you've set up the app with divshot, go to the CLI and `divshot init` an app with the same name.
+
+Install the [divshot-cli](https://github.com/divshot/divshot-cli#install) by running:
+
+`npm install divshot-cli -g`
+
+Running `divshot init` will generate a new file called `divshot.json` in the root of your project. It should look something like this:
+
+`divshot.json` example file:
+```json
+{
+  "name": "project-name",
+  "root": "dist",
+  "clean_urls": true,
+  "error_page": "error.html"
+}
+```
+
+To kick off the first deploy, type `divshot push production` to push to divshot's production environment. `divshot push` will by default push to the Development environment.
+
+### Set up some authentication 
+
+[Authentication with Divshot](http://docs.divshot.com/guides/http-authentication) is a snap.
+
+`divshot protect [environment] [username:password]`
+
+At this time it's only possible to have one set of credentials.
+
+### Set up a custom domain (optional)
+
+We like to route most of our QA through our own `sparkboxqa` domain. [Setting this up with divshot](http://docs.divshot.com/guides/domains) is pretty simple.
+
+Note that this only works for the Production environment. If you are using Divshot's promotion system you will have to use the default `staging.app-name.divshot.io` domain names.
+
+### Add some CircleCI to your life
+
+There are a few new files and configuration pieces that need to be set up before the deployments start flowing like wine. 
+
+1. **Add the project repo to CircleCI**.
+![](http://files-misc.s3.amazonaws.com/add-project.png)
+
+2. **Add divshot-cache to `.gitignore`**
+
+3. **Add necessary npm modules (e.g. grunt-cli) to `package.json`:**
+`npm install divshot-cli grunt-cli bower --save-dev`
+4. **Add a `circle.yml` file to the project**
+
+`circle.yml` contains the [configuration for CircleCI](https://circleci.com/docs/configuration). It controls things like: 
+
+  - the command to build the project
+  - how to install dependencies
+  - instructions on how to deploy the project after it's built
+
+Here is a `circle.yml` example file:
+```yml
+dependencies:
+  post:
+    - node_modules/.bin/bower install
+
+test:
+  override:
+    - node_modules/.bin/grunt dist
+
+deployment:
+  staging:
+    branch: master
+    commands:
+      - node_modules/.bin/divshot push production --token $DIVSHOT_TOKEN
```

Comment:
Do we want to default this to staging?  Or say you can leave it out and it'll go to development?